### ISSUE TITLE
@angular/common and RxJs version dependency issue

### ISSUE THREAD

**Issue by durgaprasadreddi** [ng-bootstrap/ng-bootstrap#4154]
### Bug description:
Are you planing to upgrade version of angular from 12.0.0 to 12.2.4 as we are facing dependency issue while installing on angular project which is with the version 12.2.4

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 12.2.4

ng-bootstrap: 10.0.

Bootstrap: 4.6.0
![image](https://user-images.githubusercontent.com/51472433/132300454-59f1a1c4-a55d-426a-84e3-0de56f29ed2c.png)

**Comment by pkozlowski-opensource** [comment]
Related change on the Angular side: https://github.com/angular/angular/pull/42991. Essential bit is here: https://github.com/angular/angular/pull/42991/files#diff-0b810c38f3c138a3d5e44854edefd5eb966617ca84e62f06511f60acc40546c7R14

We should probably do the same change on the ng-bootstrap side _provided_ that everything works correctly on the ng-bootstrap side with RxJS 7.0.0.

**Comment by coyoteecd** [comment]
Not sure why this is closed, I've tried upgrading rxjs in my Angular 12.2 project and `ng update` complains that ngbootstrap doesn't support it. Reopen?

**Comment by pkozlowski-opensource** [comment]
Yep, we should definitively keep it open

**Comment by initplatform** [comment]
Same issue for 11 beta 2

```bash
npm ERR! Found: rxjs@7.4.0
npm ERR! node_modules/rxjs
npm ERR!   rxjs@"7.4.0" from the root project
npm ERR!   peer rxjs@"^6.5.3 || ^7.0.0" from @angular/common@12.2.9
npm ERR!   node_modules/@angular/common
npm ERR!     @angular/common@"12.2.9" from the root project
npm ERR!     peer @angular/common@"^12.0.0" from @ng-bootstrap/ng-bootstrap@11.0.0-beta.2
npm ERR!     node_modules/@ng-bootstrap/ng-bootstrap
npm ERR!       @ng-bootstrap/ng-bootstrap@"11.0.0-beta.2" from the root project
npm ERR!     2 more (@angular/forms, @angular/platform-browser)
npm ERR!   2 more (@angular/core, @angular/forms)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer rxjs@"^6.5.5" from @ng-bootstrap/ng-bootstrap@11.0.0-beta.2
npm ERR! node_modules/@ng-bootstrap/ng-bootstrap
npm ERR!   @ng-bootstrap/ng-bootstrap@"11.0.0-beta.2" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by maxokorokov** [comment]
This is fixed with `12.0.0`, I don't think we'll patch it in `11.1`. Please reopen if necessary
